src/Periphery/ReceiverAcrossV3.sol

```diff
@@ -33,7 +33,7 @@ contract ReceiverAcrossV3 is ILiFi, TransferrableOwnership {
         address _owner,
         address _executor,
         address _spokepool
-    ) TransferrableOwnership(_owner) {
+    ) WithdrawablePeriphery(_owner) {
         owner = _owner;
```

Comment:
@ezynda3 is this setter necessary ?

Reply:
But the `owner = _owner;` is handled in there as well, so it should not be necessary here.

Reply:
I will remove `owner = _owner;` to stay consistent with the rest of the codebase.

Reply:
Fixed